<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TFDB Download manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="8" column="0" colspan="4">
     <widget class="QTextBrowser" name="download_progress_message"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Download Path:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <layout class="QGridLayout" name="mode_gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="all_checkbox">
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QProgressBar" name="download_progress_bar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="download_button">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select databases:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="browse_button">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QCheckBox" name="metadata_checkbox">
      <property name="text">
       <string>Download metadata</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="directory_line_edit">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>all_checkbox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>all_checkbox_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>download_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>download_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browse_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browse_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>all_checkbox_changed()</slot>
  <slot>download_button_clicked()</slot>
  <slot>browse_button_clicked()</slot>
 </slots>
</ui>
